src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -451,7 +451,7 @@ TvgBinCounter TvgSaver::serializeStroke(const Shape* shape, const Matrix* pTrans
 
     //width
     auto width = shape->strokeWidth();
-    if (preTransform) width *= pTransform->e11;  //we know x/y scaling factors are same.
+    if (preTransform) width *= sqrt(pow(pTransform->e11, 2) + pow(pTransform->e21, 2));  //we know x/y scaling factors are same.
```

Comment:
no i think pre-condition is missed at some points, because here xy shouldn't be different...

Reply:
I understand that the scaling in both, x and y axis, are the same
but element e11 contains informations about the scaling and the rotation, that why we can not use only this value

Reply:
aha got it!